### DISCUSSION TITLE
Is there a plan to add the ability to customize the mahjong board? Or is there a place to generate mahjong codes?

### DISCUSSION THREAD

**Discussion by AnEht** [Equim-chan/mjai-reviewer#108] [Ideas]
I to this may bring the cost of cheating down further, but everything is a double-edged sword. Are there plans to add the ability to customize mahjong games? Or is there a place to generate mahjong codes?

**Comment by AnEht** [comment]
（自定义麻将牌局）（局面代码）

**Comment by Equim-chan** [comment]
https://tenhou.net/6
